### PR TITLE
Added Calendar input with date/time Picker

### PR DESCRIPTION
**Pull request by iiLubos**
MMCalendarEditor {
          title: "MMCalendarEditor"
          placeholderText: "YYYY/MM/DD"
          onSelected: function(dateTime) { text = dateTime }
        }

### CODE REVIEW

File: app/qml/inputs/MMIconInputEditor.qml
Comment (tomasMizera) [inline app/qml/inputs/MMIconInputEditor.qml:1]: Let's make this qrcode/calendar instead of another generic component

**Review by tomasMizera — CHANGES_REQUESTED** [review]
Create qrcode and calendar inputs

File: app/qml/components/MMAmPmSwitch.qml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Controls.Basic
+
+// Prepared switch, but to used yet
```
Comment (PeterPetrik) [inline app/qml/components/MMAmPmSwitch.qml:14]: typo?

File: app/qml/components/MMAmPmSwitch.qml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+/***************************************************************************
```
Comment (PeterPetrik) [inline app/qml/components/MMAmPmSwitch.qml:1]: wondering if not to put all "sub" components to subfolder (`components/calendar`) and just keep the calendar that can be used from rest of the application in `comonents` folder?

**Review by tomasMizera — COMMENTED** [review]
Minor comments :)

**Review by tomasMizera — APPROVED** [review]

**Pull request merged**